frontend/webadmin/modules/frontend/src/main/resources/org/ovirt/engine/ui/frontend/AppErrors.properties

```diff
@@ -1635,6 +1635,8 @@ VALIDATION_GLUSTER_VOLUME_OPTION_KEY_NOT_NULL=Option Key is required.
 VM_SLA_POLICY_UNCHANGED=VM SLA Policy command does not change anything.
 ACTION_TYPE_FAILED_SERVER_PART_OF_ANOTHER_CLUSTER=Cannot ${action} ${type}. Server ${server} is already part of another cluster.
 VALIDATION_VM_MIGRATION_DOWNTIME_RANGE=Migration downtime must be between 0 and 2147483647 milliseconds.
+VALIDATION_PARALLEL_MIGRATIONS_RANGE=Custom number of parallel migration connections must be between 2 and 255.
+VALIDATION_PARALLEL_MIGRATIONS_CLUSTER=Cluster cannot inherit parallel migration settings and custom number of parallel migration connections must be between 2 and 255.
```

Comment:
wouldn't it be nicer to return a different message in the cluster-validator based on whether the input is null or out-of-range?

Reply:
It would, done.